datasets/_wikidata/categories/wd_categories.yml

```diff
@@ -845,6 +845,522 @@ config:
+      country: co
+      topic: role.pep
+      position:
+        name: President of Colombia
+        country: co
+        topics:
+          - gov.head
+          - gov.national
+    - category: Government_ministers_of_Colombia
+      country: co
+      topic: role.pep
+      position:
+        name: Minister of Colombia
+        country: co
+        topics:
+          - gov.executive
+          - gov.national
+    - category: Mayors_of_Bogotá
+      country: co
+      topic: role.pep
+      position:
+        name: Mayor of Bogotá
+        country: co
+        topics:
+          - gov.muni
+          - gov.head
+    - category: Lists_of_political_office-holders_in_Colombia
+      country: co
+      topic: role.pep
+    - category: Senadores_de_Colombia
+      country: co
+      language: es
+      topic: role.pep
+      position:
+        name: Senator
+        country: co
+        topics:
+          - gov.legislative
+          - gov.national
+    - category: Diputados_de_Colombia
+      country: co
+      language: es
+      topic: role.pep
+    - category: Presidentes_de_Colombia
+      country: co
+      language: es
+      topic: role.pep
+      position:
+        name: President of Colombia
+        country: co
+        topics:
+          - gov.head
+          - gov.national
+    - category: Vicepresidencia_de_Colombia
+      country: co
+      language: es
+      topic: role.pep
+      position:
+        name: Vice President of Colombia
+        country: co
+        topics:
+          - gov.executive
+          - gov.national
+    - category: Políticos_de_Colombia
+      country: co
+      language: es
+      topic: role.pep
+
+    # Venezuela
+    - category: Members_of_the_National_Assembly_(Venezuela)
+      country: ve
+      topic: role.pep
+      position:
+        name: Member of the National Assembly
+        country: ve
+        topics:
+          - gov.legislative
+          - gov.national
+    - category: Members_of_the_Senate_of_Venezuela
+      country: ve
+      topic: role.pep
+      position:
+        name: Member of the Senate of Venezuela
+        country: ve
+        topics:
+          - gov.legislative
+          - gov.national
+    - category: Presidents_of_Venezuela
+      country: ve
+      topic: role.pep
+      position:
+        name: President of Venezuela
+        country: ve
+        topics:
+          - gov.head
+          - gov.national
+    - category: Vice_presidents_of_Venezuela
+      country: ve
+      topic: role.pep
+      position:
+        name: Vice President of Venezuela
+        country: ve
+        topics:
+          - gov.executive
+          - gov.national
+    - category: Government_ministers_of_Venezuela
+      country: ve
+      topic: role.pep
+      position:
+        name: Minister of Venezuela
+        country: ve
+        topics:
+          - gov.executive
+          - gov.national
+    - category: Venezuelan_politicians
+      country: ve
+      topic: role.pep
```

Comment:
I would tag "office-holders" as PEP, but not "Venezuelan politicians" (and analogous for other countries)